etc/jenkins/docker/pom.xml

```diff
@@ -0,0 +1,296 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+    Copyright (c) 2022 Oracle and/or its affiliates. All rights reserved.
+    This program and the accompanying materials are made available under the
+    terms of the Eclipse Public License v. 2.0 which is available at
+    http://www.eclipse.org/legal/epl-2.0,
+    or the Eclipse Distribution License v. 1.0 which is available at
+    http://www.eclipse.org/org/documents/edl-v10.php.
+    SPDX-License-Identifier: EPL-2.0 OR BSD-3-Clause
+-->
+
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd"
+         xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+    <modelVersion>4.0.0</modelVersion>
+
+    <name>EclipseLink Docker Image</name>
+    <description>Docker image used in Eclipse Foundation build infrastructure to build EclipseLink and run LRG tests</description>
+    <groupId>org.eclipse.persistence.build</groupId>
+    <artifactId>org.eclipse.persistence.build.docker</artifactId>
+    <!-- Version is used as docker image tag -->
+    <version>2.0.0-EA1</version>
+
+    <properties>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <docker.image>tkraus/el-build</docker.image>
+        <docker.source>oraclelinux:8</docker.source>
+        <java.pkg>openjdk-17.0.2_linux-x64_bin.tar.gz</java.pkg>
+        <java.url>https://download.java.net/java/GA/jdk17.0.2/dfd4a8d0985749f896bed50d7138ee7f/8/GPL/${java.pkg}</java.url>
```

Comment:
What is this? Why not to use https://www.oracle.com/java/technologies/jdk-script-friendly-urls/ ?

Reply:
fixed.